tensorflow_datasets/core/visualization.py

```diff
@@ -85,46 +85,82 @@ def show_examples(ds_info, ds, rows=3, cols=3, plot_scale=3., image_key=None):
 
     image_key = image_keys[0]
```

Comment:
I think check first like` if image_keys` then only set `image_key `as if data is not image dataset `image_key` set to empty

Reply:
Refer to comment 78 and code 79 - The check to run this code has been added ( only when the image instances are populated )

Reply:
Ohh sorry its my mistake